specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2019-08-01-preview/OperationalInsights.json

```diff
@@ -0,0 +1,429 @@
+      }
+    }
+  },
+  "definitions": {
+    "DataExport": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DataExportProperties",
+          "description": "data export properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "description": "The top level data export resource container.",
+      "x-ms-azure-resource": true
+    },
+    "DataExportListResult": {
+      "description": "Result of the request to list data exports.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataExport"
+          },
+          "description": "List of data export instances within a workspace.."
+        }
+      }
+    },
+    "DataExportProperties": {
+      "properties": {
+        "dataExportId": {
+          "type": "string",
+          "description": "The data export rule ID."
+        },
+        "allTables": {
+          "type": "boolean",
+          "description": "When ‘true’, all workspace's tables are exported."
+        },
+        "tableNames": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "An array of tables to export, for example: [“Heartbeat, SecurityEvent”]."
+        },
+        "destination": {
+          "$ref": "#/definitions/Destination",
+          "description": "destination properties.",
+          "x-ms-client-flatten": true
+        },
+        "enable": {
+          "type": "boolean",
+          "description": "The data export rule creation time."
+        },
+        "createdDate": {
+          "type": "string",
+          "description": "The latest data export rule modification time."
+        },
+        "lastModifiedDate": {
+          "type": "string",
+          "description": "Date and time when the export was last modified."
+        }
+      },
+      "description": "Data Export properties."
+    },
+    "Destination": {
+      "properties": {
+        "resourceId": {
+          "type": "string",
+          "description": "The destination resource ID. This can be copied from the Properties entry of the destination resource in Azure."
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of the resource",
+          "enum": [
+            "StorageAccount",
+            "EventHub"
+          ]
+        },
+        "metaData": {
+          "$ref": "#/definitions/DestinationMetaData",
+          "description": "destination meta data.",
+          "x-ms-client-flatten": true
+        }
+      },
+      "required": [
+        "resourceId",
+        "type"
+      ],
+      "description": "Destination properties."
+    },
+    "DestinationMetaData": {
+      "properties": {
+        "eventHubName": {
+          "type": "string",
+          "description": "event hub name."
```

Comment:
isn't this evident from the resourceId provided .. ?

Reply:
No, It's Event Hubs namespace. The namespace name is  what we need the eventHubName is optional.